### DISCUSSION TITLE
Import Playlists AddOn generates errors

### DISCUSSION THREAD

**Discussion by Fixu69** [rern/rAudio#187] [Fixed Issues] [answered]
Hi,

I am new with rAudio but used RuneAudio long time. 
Now I would like to import my playlists. So I copied them to /var/lib/mpd/playlists.
But the playlists I used without problems on RuneAudio generates now errors saying "error adding NAS/MP3s/folder/file.mp3
: No such directory"

although I can list the mp3-files with command ls /mnt/MPD/NAS/MP3s
The format in the playlist is one file path per line like: NAS/MP3s/folder/file.mp3

Is this not how it should work?

Thanks in advance for your help!

**Comment by rern** [comment, marked as answer]
Try:
`mpc ls NAS/MP3s`

If `MPD error: No such directory`, update Library before import.

**Reply by Fixu69** [reply]
mpc ls NAS/MP3s - lists all the folders how it should.

I started "Updates for rAudio 1" and than again the import but no luck:

wget -q https://github.com/rern/rAudio-addons/raw/main/playlist/importplaylist.sh
chmod 755 importplaylist.sh
./importplaylist.sh plsi Import main 

 .   Import Playlists .

 .  Playlist Import .

Testliste
error adding NAS/MP3s/All4mambo/Boom Boom Boom.mp3
: No such directory
-1

Duration 00:01

 .  Done.

>NAS/MP3s/All4mambo/Boom Boom Boom.mp3< is the fiirst line of about 14 in the Testliste.m3u

**Reply by rern** [reply]
If it was edited on Windows, verify line endings are `LF`, not `CR` `LF`.
- Get [Notepad++](https://notepad-plus-plus.org/).
- Edit > EOL conversion > Unix (LF)

**Reply by Fixu69** [reply]
what a beginner mistake. tststs...

Now, with only (LF) as lines brake it is working fine!
Exept one playlist, it says the one is too long ;-) what is the limit of mp3 files per playlist?

Thank you for your prompt help and your great rAudio!
Best regards
